paper/paper.md

```diff
@@ -0,0 +1,203 @@
+This allows finding solutions to more complex problems such as anisotropic materials, active media or magneto-optic samples.
+
+**PyElli** ensures fast processing through fully vectorized algorithms for multiple wavelengths and by leveraging numerical algebra libraries like [NumPy](https://numpy.org) [@harris2020array] and [SciPy](https://scipy.org) [@2020SciPy-NMeth].
+Together, these runtime advantages enable the practical use of advanced fitting algorithms such as global optimizers while maintaining reasonable evaluation times.
+As a result, they enable integrated in-situ monitoring and real-time data analysis of overlayer growth.
+Furthermore, the use of Python and vectorization libraries also facilitates the development of artificial intelligence-based SE data analysis.
+
+# Statement of need
+
+The importance of publishing data according to the FAIR principles is growing [@Wilkinson2016].
+Many research journals already require authors to add supporting data, and there is a growing expectation from sponsors for institutes and researchers to implement data governance.
+The FAIR principles have recently been extended to apply to research software as well since reproducing data requires not only the data itself but also the software used to create it [@Barker2022].
+Producing FAIR data and using a FAIR and open analysis pipeline is especially important for SE, as the results are tightly related and dependent on the algorithms and models used for evaluation.
+
+An open-source toolkit has many inherent benefits over proprietary software.
+For SE, optical models vary between manufacturers and translation can be difficult without comprehensive documentation.
+**PyElli's** open-source nature makes optical models extendable, auditable, and fully comprehensive.
+Each version of **PyElli** is associated with a DOI and a Zenodo upload, allowing for reliable referencing and reproducibility of analysis results.
+It supports [NXellipsometry](https://fairmat-experimental.github.io/nexus-fairmat-proposal/9636feecb79bb32b828b1a9804269573256d7696/classes/contributed_definitions/NXellipsometry.html#nxellipsometry), a recent advancement in SE data standardization.
+This standard is developed by the German consortium [FAIRmat](https://www.fair-di.eu/fairmat) within the German National Research Data Infrastructure (NFDI) and is accepted by the [NeXus International Advisory Committee (NIAC)](https://www.nexusformat.org/NIAC.html), which has been maintaining beamline standards in the [NeXus format](https://www.nexusformat.org) for over 20 years [@Konnecke2015].
+**PyElli** is also available as an [example](https://gitlab.mpcdf.mpg.de/nomad-lab/nomad-remote-tools-hub/-/tree/develop/docker/ellips/example) and a cloud-based [analysis tool](https://nomad-lab.eu/prod/v1/staging/gui/analyze/north) in the research data management software [NOMAD](https://nomad-lab.eu/nomad-lab/) [@Scheffler2022].
```

Comment:
The example also changed places and is now here: https://gitlab.mpcdf.mpg.de/nomad-lab/north/ellips/-/tree/main/example?ref_type=heads